kubejs/data/bountiful/bounties/pools/resources/metal_rare.json

```diff
@@ -0,0 +1,17 @@
+{
+    "modsRequired":[],
+    "content":[
+        { "type": "item", "content": "minecraft:gold_ingot", "amountRange": {"min": 8, "max": 32}, "unitWorth": 300, "weight":100},
+        { "type": "item", "content": "minecraft:iron_ingot", "amountRange": {"min": 8, "max": 32}, "unitWorth": 300, "weight":100},
+        { "type": "item", "content": "emendatusenigmatica:copper_ingot", "amountRange": {"min": 8, "max": 32}, "unitWorth": 300, "weight":100},
+        { "type": "item", "content": "emendatusenigmatica:aluminum_ingot", "amountRange": {"min": 8, "max": 32}, "unitWorth": 300, "weight":100},
+        { "type": "item", "content": "emendatusenigmatica:silver_ingot", "amountRange": {"min": 8, "max": 32}, "unitWorth": 300, "weight":100},
+        { "type": "item", "content": "emendatusenigmatica:lead_ingot", "amountRange": {"min": 8, "max": 32}, "unitWorth": 300, "weight":100},
+        { "type": "item", "content": "emendatusenigmatica:nickel_ingot", "amountRange": {"min": 8, "max": 32}, "unitWorth": 300, "weight":100},
+        { "type": "item", "content": "emendatusenigmatica:uranium_ingot", "amountRange": {"min": 8, "max": 32}, "unitWorth": 300, "weight":100},
+        { "type": "item", "content": "emendatusenigmatica:osmium_ingot", "amountRange": {"min": 8, "max": 32}, "unitWorth": 300, "weight":100},
+        { "type": "item", "content": "emendatusenigmatica:tin_ingot", "amountRange": {"min": 8, "max": 32}, "unitWorth": 300, "weight":100},
+        { "type": "item", "content": "emendatusenigmatica:zinc_ingot", "amountRange": {"min": 8, "max": 32}, "unitWorth": 300, "weight":100},
+        { "type": "item", "content": "emendatusenigmatica:iridium_ingot", "amountRange": {"min": 8, "max": 32}, "unitWorth": 900, "weight":50}
```

Comment:
We don't actually use iridium for anything. It's just a stock part of EE

Reply:
Too much E2E knowledge floating about \o/

Reply:
:D

Reply:
How's this going to add in to bountiful?

Reply:
The idea is to also add in extra decrees that can make use of these new loot tables.
Like a Miner, who you'd give a pickaxe and some torches.